pyproject.toml

```diff
@@ -37,6 +37,9 @@ dynamic = ["version"]
 [project.license]
 file = "LICENSE"
 
+[project.optional-dependencies]
+marimo = ["marimo>=0.18.4"]
+
```

Comment:
no need to declare it as an optional dependency either, move to `tests` under `[dependency-groups]` below